README.md

```diff
@@ -14,68 +14,56 @@ files for assignments that require GatorGradle. ConfiGator uses
 
 ## Usage
 
-### Installing Python Dependencies
+### Installing ConfiGator
 
-In order to install the dependencies of this project, run this command in your
-command line:
+Command to install ConfiGator
 
 ```bash
-poetry install
-```
-
-```bash
-poetry install typer
-```
-
-After installing the dependencies, navigate to the repository directory, and run
-the following in your command line:
-
-```bash
-poetry run python main.py
+pipx install configator
 ```
 
 ### Commands to Change Specific Information Within a File
 
 Command to change name
 
 ```bash
-poetry run python main.py --name {your name}
+ configator --name {your name}
 ```
 
 Command to change break format
 
 ```bash
-poetry run python main.py --brk {false}
+configator --brk {false}
 ```
 
 Command to change fast fail
 
 ```bash
-poetry run python main.py --ff {true}
+configator --ff {true}
 ```
 
 Command to change indentation
 
 ```bash
-poetry run python main.py --ind {desired int}
+configator --ind {desired int}
 ```
 
 Command to change GatorGrader version
 
 ```bash
-poetry run python main.py --vers {desired version}
+configator --vers {desired version}
 ```
 
 Command to change GatorGradle
 
 ```bash
-poetry run python main.py --ggradleversion {desired version}
+configator --ggradleversion {desired version}
 ```
 
 Command to change commit requirements
 
 ```bash
-poetry run python main.py --com {desired int}
+configator --com {desired int}
 ```
```

Comment:
These commands should not be listed out here; instead, point the user to `configator --help`, which should list everything available (and will also fix errors like `--com` no longer being an option)